scripts/st2client-startup.sh

```diff
@@ -14,3 +17,6 @@ while true; do
     sleep infinity
   fi
 done
+
+echo "No packs were able to be registered due to st2 failures. You may need to load them manually"
```

Comment:
Let's make the message more explicit here:

```suggestion
echo "Error! No packs were able to be registered due to st2 connect failures! You may need to load them manually."
```